compiler/prelude/prelude.go

```diff
@@ -335,6 +335,10 @@ var $append = function(slice) {
 };
 
 var $appendSlice = function(slice, toAppend) {
+  if (toAppend.constructor === String) {
+    var bytes = $stringToBytes(toAppend);
+    return $internalAppend(slice, bytes, 0, bytes.length);
+  }
```

Comment:
Question, why was this seemingly moved from Go code in `expressions.go` to here?